docs/deploy/gcp.md

```diff
@@ -59,21 +59,71 @@ For example, if you install lakeFS on GKE, you need to deploy the SQL Auth Proxy
    **Note:** it is preferable to run the binary as a service using systemd or your operating system's facilities.
 
 ### On Google Cloud Run
-To support container-based environments like Google Cloud Run, lakeFS can be configured using environment variables. Here is a `docker run`
-command to demonstrate starting lakeFS using Docker:
-
-```sh
-docker run \
-  --name lakefs \
-  -p 8000:8000 \
-  -e LAKEFS_DATABASE_TYPE="postgres" \
-  -e LAKEFS_DATABASE_POSTGRES_CONNECTION_STRING="[DATABASE_CONNECTION_STRING]" \
-  -e LAKEFS_AUTH_ENCRYPT_SECRET_KEY="[ENCRYPTION_SECRET_KEY]" \
-  -e LAKEFS_BLOCKSTORE_TYPE="gs" \
-  treeverse/lakefs:latest run
-```
-
-See the [reference](../reference/configuration.md#using-environment-variables) for a complete list of environment variables.
+To support container-based environments like Google Cloud Run, lakeFS can be configured using environment variables.
+
+1. Save the following configuration file as `config.yaml`:
+
+   ```yaml
+   apiVersion: serving.knative.dev/v1
+   kind: Service
+   metadata:
+     name: [LAKEFS_SERVICE_NAME]
+   spec:
+     template:
+       metadata:
+         annotations:
+           autoscaling.knative.dev/maxScale: '5'
+           run.googleapis.com/client-name: cloud-console
+           run.googleapis.com/cloudsql-instances: [POSTGRES_INSTANCE_NAME]
+       spec:
+         containers:
+         - image: gcr.io/lakefs/treeverse/lakefs@sha256:18be2f7e86f7572160909b7dcae2e50be5704cb174eeaf4fb61274def3536009
+           ports:
+           - name: http1
+             containerPort: 8000
+           env:
+           - name: LAKEFS_BLOCKSTORE_TYPE
+             value: gs
+           - name: LAKEFS_BLOCKSTORE_GS_CREDENTIALS_JSON
+             value: '[YOUR SERVICE ACCOUNT JSON STRING]'
```

Comment:
Use secrets or other mechanism to pas the credentials - but not environment vars.

Reply:
@nopcoder
Should it be like this:
           - name: LAKEFS_BLOCKSTORE_GS_CREDENTIALS_JSON
             valueFrom:
               secretKeyRef:
                 name: [YOUR_SERVICE_ACCOUNT_JSON_STRING_SECRET_KEY]

Also, do we have to provide gcloud CLI command to create secrets or should we leave up to customer's GCP admin to handle this?

Reply:
@kesarwam This is the syntax (missing 'key' property). But, you will need to instruct (documentation or commands) to add the value of the service account as secret. So, the name as the value of '[YOUR_SERVICE_ACCOUNT_JSON_STRING_SECRET_KEY]' is not right. We like to pull it from the secrets.